common/getcommands.c

```diff
@@ -47,12 +40,6 @@ TSS2_RC GetCommands( TSS2_SYS_CONTEXT *resMgrSysContext, TPML_CCA **supportedCom
     TSS2_RC rval = TSS2_RC_SUCCESS;
     UINT32 i;
     
-#ifdef SAPI_CLIENT
-    void *(*gcMalloc)(size_t size) = malloc;
-#else    
-    void *(*gcMalloc)(size_t size) = rmMalloc;
```

Comment:
Problem here is that this code is used by RM, too, and we needed to return a RM-specific error level.  Unless you accounted for that in the RM code, this error level won't come out right.